### ISSUE TITLE
Consider a more generic name

### ISSUE THREAD

**Issue by willmtemple** [VoLuong/Awesome-Ubuntu-Linux#35]
The inclusion of "Ubuntu" specifically in the repository title is confusing, and implies that the list is *only* for Ubuntu, when it is for users of Linux distributions, generally.

A very small number of the included applications are Ubuntu-specific, such as the Ubuntu SDK. The large majority are available on any linux distribution.

Other distributions (elementaryOS, Mint) are beginning to take a larger portion of the desktop linux share. Mint is even ranked higher than Ubuntu on distrowatch.org (not exactly a scientific source, but all the same, some indicator of interest). Also, many of us use other distributions such as Fedora, Arch, Debian, Slackware, Gentoo, CentOS, SUSE, Manjaro, and literally hundreds of others.

Perhaps it would be pertinent to create an "Ubuntu" category for any Ubuntu-specific software and name this repository "awesome-linux-software".

**Comment by VoLuong** [comment]
Thank you for the issue. 
Well, I am sorry if that makes you feel confusing . Actually the name is Awesome Ubuntu/Linux ( it means ubuntu or linux in general).  I didnt mean that this list is only for ubuntu. Putting ubuntu in the name is like , well, declare a specific example. I could have named it Awesome-LinuxMint-Linux or Awesome-ElementaryOS-Linux but I think it doesnt matter at all. There was no bias inside the list or the name (You can check it yourself :wink: ).  I myself used a lot of distros and currently stick with Ubuntu , but that doesnt mean this list is promoting Ubuntu or giving any specific advantages to ubuntu user. Plus, new users often find it friendlier to access with Ubuntu in the name, regardless of the fact that he or she is another distro's user. 
In short, it is just a name. 
I totally understood your concern, but in my opinion the content and the title is clear enough to show the purpose of this repo. 
The only thing that is exclusively for ubuntu is the set up part, which i am leaving other distros' parts : to be added later. If you can gather some source , I highly recommend contributing in that section :tada: 
Thank you. I will close this issue for now but feel free to discuss more :+1:

**Comment by VoLuong** [comment]
> Names, I think, are surprisingly important

yes i agreed but i didnt think it was **that important**.

>  If this repository were named Awesome-Arch-Linux it would rightly alienate users of any other Linux distribution

I agreed with this too, that 's why previously I said 

>  Plus, new users often find it friendlier to access with Ubuntu in the name (feeling)

I wanted to make it friendly with new users so I picked one of the most used distro as an example.  Other pro users will focus more on the linux letter and they often tend to explore stuffs themselves to see what i am writing here, so there's no worry about it. 

And about :

> To be fair, the name of the repository is not Awesome Ubuntu/Linux

Yes it is :< . Look at the bold one, that's not a title only :< but also a true name.
![](http://i.imgur.com/bWzgJAj.png) 
What you are seeing is because of this 
![image](https://cloud.githubusercontent.com/assets/15828926/17593590/9df175b6-6010-11e6-8802-0eb92a3c29be.png)

> When a repo is named "Ubuntu", I assume that it has something to do with the Ubuntu ecosystem, e.g. Unity, Snaps, Mir, PPAs, obscure glib bugs that were patched years ago

Sorry for this. I didnt see this coming. I was just on github for about 1 year and never went by any of these stuffs. So sorry. But I dont think your assumption is the same assumption that everyone will have.
And to express more about why I didnt include the word "software" in the name, I think the word "Awesome" is better. It is more descriptive and interesting. You find something awesome when you kinow what you can do with it , how you can utilize something , etc.
If this is about Unity, Snaps, Mir, PPAs... I would have at least put 1 of them in the name. And if this is only about Ubuntu why would I have include the "linux" in the name . It is so lengthy and redundant right ?

> If I had been looking for a generic Linux resource, I probably wouldn't have ended up here.

Sorry, again.

**Comment by willmtemple** [comment]
I'm aware that github doesn't allow slashes in repository names (technical limitation). All the same, the title at the top of the README is not the name of the repo, it's just a piece of text in the README.

![screenshot from 2016-08-11 15-20-09](https://cloud.githubusercontent.com/assets/666532/17601731/8c324f50-5fd7-11e6-81ef-4257bc4ab8ad.png)

The description helps to clarify a little bit, but I think most people read only the names and then the descriptions if the name isn't clear. I also think that "tools and other materials for Linux users" is better than "tools and other materials for Ubuntu/Linux." In this case, the name *is* clear, it's just not exactly accurate.

>I wanted to make it friendly with new users so I picked one of the most used distro as an example. Other pro users will focus more on the linux letter and they often tend to explore stuffs themselves to see what i am writing here, so there's no worry about it.

But ubuntu may not remain the most popular. And I think that you may be mistaken here. In my experience, moderate-to-advanced users of any non-ubuntu distribution learn very quickly that Ubuntu resources (1) are inaccurate for all distros and (2) tend to be highly dated. Ubuntu is the most "special snowflake" distro that exists and it's unlikely that tweaks for ubuntu will work on others. I say this as a 10-year user (shorter than many but longer than most, I think) of GNU/Linux distributions that also started with Ubuntu back in 2006.

> I think the word "Awesome" is better. It is more descriptive and interesting. You find something awesome when you kinow what you can do with it , how you can utilize something , etc.

I'm not sure what this means, honestly. `Awesome` is not descriptive. It's a feelgood github convention at best. All it says is that this is a particular kind of repository that curates a list of links to other works. Your other repo `awesome-courses` illustrates that this has nothing to do with software.

> If this is about Unity, Snaps, Mir, PPAs... I would have at least put 1 of them in the name.

That... wouldn't make any sense. If it was a repo about ubuntu ecosystem tweaks and you named the repo `awesome-snaps-ubuntu`, then the title suggests the repo is *just* about snaps when in fact it's about Ubuntu generally, so then it would be appropriate to call it `awesome-ubuntu`.

> And if this is only about Ubuntu why would I have include the "linux" in the name.

Exactly. It's confusing whichever way you look at it. If it's for Linux generally then I would think it should be `awesome-linux` or `awesome-linux-software`. If it's for Ubuntu specifically then I would think it should be `awesome-ubuntu` or `awesome-ubuntu-software`.

Anyway. This clearly hasn't been a barrier to the exposure of this repository. Clearly many users are enjoying it (100+ stars yesterday!). But, it's very troubling that the word "Ubuntu" is becoming synonymous with "Linux." This is dangerous, because Canonical has a history of pushing Ubuntu in an anti-community direction. Free software is best developed in communities of upstreams with cooperation from corporations, but Canonical has continually eschewed this idea. The community builds an alternative display protocol (Wayland), and Canonical subverts the community by developing a (pointless) second alternative (Mir) that only they will use. SystemD builds a new init manager to replace SysV init, adopted by Fedora, Arch, Debian, and even CentOS/RHEL at this point. Canonical builds upstart, which is a totally incompatible init system (that they have _already_ abandoned on the newer versions). The community builds a specification for containerized apps (flatpak), and Canonical not only develops its competing technology (snaps) but has the audacity to claim publicly that snaps will be supported on all major linux distributions (they didn't bother to consult with those other distributions about this decision).

All this does is fragment us as users even more. Canonical (ubuntu) is the most serial offender in this regard. And it's deeply frustrating that more frequently we see "Ubuntu" touted as a synonym to "Linux", which it unequivocally is not. I would even argue that they are becoming more divergent as Canonical tries to make Ubuntu's identity unique, and as they have actively worked against emergent community efforts to the detriment of the linux ecosystem at large.

**Comment by VoLuong** [comment]
> But ubuntu may not remain the most popular. And I think that you may be mistaken here. In my experience, moderate-to-advanced users of any non-ubuntu distribution learn very quickly that Ubuntu resources (1) are inaccurate for all distros and (2) tend to be highly dated. Ubuntu is the most "special snowflake" distro that exists and it's unlikely that tweaks for ubuntu will work on others. I say this as a 10-year user (shorter than many but longer than most, I think) of GNU/Linux distributions that also started with Ubuntu back in 2006.

Ok! Now this makes sense to me. I will consider changing the repo's name in the next 24 hour. Thank you very much for taking time explaing this to me.

> But, it's very troubling that the word "Ubuntu" is becoming synonymous with "Linux." This is dangerous, because Canonical has a history of pushing Ubuntu in an anti-community direction.

Ok, I understand.

**Comment by VoLuong** [comment]
Name changed now. Thank you so much. And sorry for taking so long to fully understand your opinion.

**Comment by VoLuong** [comment]
Is the name fine now :-?

**Comment by willmtemple** [comment]
Thank you! I think that this has been a very productive discussion. And thanks much for understanding. I know that it seems to be a very nitpicky issue, but this is important to a lot of us!

The one remark I would make is that `Awesome-Linux-Softwares` should probably be `Awesome-Linux-Software`, but that's a super small niptick.

Cheers!

**Comment by VoLuong** [comment]
Cheers! 
All for the sake of Linux community.
